AzureMonitorAgent/ama_tst/modules/logcollector.py

```diff
@@ -81,11 +81,58 @@ def copy_dircontents(src, dst):
 
 # Log collecting functions
 
+def collect_mdsd_environ(output_dirpath):
```

Comment:
can make this a generic function that takes process name as argument and we can potentially collect for both mdsd and amacoreagent